_posts/2019-08-26-how-to-use-gruntwork-infrastructure-as-code-library.adoc

```diff
@@ -797,261 +781,115 @@ Versioned URL::
-}
+Familiarize yourself with the output variables in https://github.com/gruntwork-io/terraform-aws-service-catalog/blob/master/modules/networking/vpc/outputs.tf[`outputs.tf`]:
 
-output "private_persistence_subnet_cidr_blocks" {
-  description = "The CIDR blocks of the private persistence subnets"
-  value       = module.vpc.private_persistence_subnet_cidr_blocks
-}
-
-output "public_subnet_ids" {
-  description = "The IDs of the public subnets"
-  value       = module.vpc.public_subnet_ids
-}
-
-output "private_app_subnet_ids" {
-  description = "The IDs of the private app subnets"
-  value       = module.vpc.private_app_subnet_ids
-}
-
-output "private_persistence_subnet_ids" {
-  description = "The IDs of the private persistence subnets"
-  value       = module.vpc.private_persistence_subnet_ids
-}
-----
-
-[[manual_tests_terraform]]
-==== Manual tests for Terraform code
-
-Now that the code is written, you may want to test it manually. We recommend testing in a _sandbox environment_ where
-you can deploy infrastructure without affecting any other environments (especially production!). For example, if you're
-using AWS, this should be a separate AWS account.
-
-The easiest way to test is to create a `testing/terraform.tfvars` file:
-
-----
-infrastructure-modules
-  └ networking
-    └ vpc-app
-      └ main.tf
-      └ outputs.tf
-      └ variables.tf
-      └ testing
-        └ terraform.tfvars
-----
-
-Inside this file, you can set all the variables for your module to test-friendly values:
-
-.infrastructure-modules/networking/vpc-app/testing/terraform.tfvars
-[source,hcl]
-----
-aws_region       = "us-east-2"
-aws_account_id   = "555566667777"
-vpc_name         = "example-vpc"
-cidr_block       = "10.0.0.0/16"
-num_nat_gateways = 1
-----
-
-You should also add a `testing/backend.hcl` file:
-
-----
-infrastructure-modules
-  └ networking
-    └ vpc-app
-      └ main.tf
-      └ outputs.tf
-      └ variables.tf
-      └ testing
-        └ terraform.tfvars
-        └ backend.hcl
-----
-
-In this file, you can configure test-friendly settings for your backend. For example, if you're using the S3 backend,
-you can specify:
-
-.infrastructure-modules/networking/vpc-app/testing/backend.hcl
-[source,hcl]
-----
-bucket = "<YOUR-BUCKET-FOR-TESTING>"
-key    = "manual-testing/<YOUR-NAME>/terraform.tfstate"
-region = "us-east-2"
-----
-
-You can now test manually by authenticating to your sandbox environment (see
-https://blog.gruntwork.io/a-comprehensive-guide-to-authenticating-to-aws-on-the-command-line-63656a686799[A Comprehensive Guide to Authenticating to AWS on the Command Line])
-and running:
-
-[source,bash]
-----
-cd infrastructure-modules/networking/vpc-app/testing
-terraform init -backend-config=backend.hcl ../
-terraform apply ../
-----
-
-When you're done testing, clean up by running:
-
-[source,bash]
-----
-terraform destroy ../
-----
 
 [[automated_tests_terraform]]
 ==== Automated tests for Terraform code
```

Comment:
Like above, this section is a little bit confusing. It is useful as informational purposes, but is only relevant to users who would be building their own module, or will be contributing code.

Perhaps we can streamline this section to focus entirely on what a consumer of the service catalog would need to do, and separate the contribution question to another part of the docs?